code/game/jobs/access_datum.dm

```diff
@@ -298,7 +298,15 @@
 	region = ACCESS_REGION_SUPPLY
 
 // /var/const/free_access_id = 51
-// /var/const/free_access_id= 52
+
+//YW ADDITION START

```

Comment:
could this datum pleased be moved to its own file aka access_datum_yw

Reply:
Good idea. I'll give it a shot in a few minutes; currently running tests for a fix to #600.

Reply:
Done.